mobile_rewards/src/main.rs

```diff
@@ -1,43 +1,77 @@
-use mobile_rewards::{mk_db_pool, Result, Server};
+use clap::Parser;
+use mobile_rewards::{Result, Server as MobileServer, Settings};
+use std::path;
 use tokio::signal;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 
-#[tokio::main]
-async fn main() -> Result {
-    dotenv::dotenv()?;
-    tracing_subscriber::registry()
-        .with(tracing_subscriber::EnvFilter::new(
-            std::env::var("RUST_LOG").unwrap_or_else(|_| "mobile_rewards=debug".into()),
-        ))
-        .with(tracing_subscriber::fmt::layer())
-        .init();
-
-    // Install the prometheus metrics exporter
-    poc_metrics::install_metrics();
-
-    // Create database pool
-    let pool = mk_db_pool(10).await?;
-    sqlx::migrate!().run(&pool).await?;
-
-    // Configure shutdown trigger
-    let (shutdown_trigger, shutdown_listener) = triggered::trigger();
-    tokio::spawn(async move {
-        let _ = signal::ctrl_c().await;
-        shutdown_trigger.trigger()
-    });
-
-    // Reward server keypair from env
-    let keypair_file = std::env::var("REWARD_SERVER_KEYPAIR")?;
-    let rs_keypair = load_from_file(&keypair_file)?;
-
-    // Reward server
-    let mut reward_server = Server::new(pool.clone(), rs_keypair).await?;
-
-    reward_server.run(shutdown_listener.clone()).await?;
-    Ok(())
+#[derive(Debug, clap::Parser)]
+#[clap(version = env!("CARGO_PKG_VERSION"))]
+#[clap(about = "Helium Mobile Rewards Server")]
```

Comment:
This value should appear in Cargo.toml for one place to maintain.
```suggestion
#[clap(about = env!("CARGO_PKG_DESCRIPTION"))]
```
See https://doc.rust-lang.org/cargo/reference/environment-variables.html